src/data/i18n/locales/whitelist_fr.json

```diff
@@ -0,0 +1,2 @@
+[
+]
```

Comment:
What's the purpose of this file?

Reply:
output from the `i18n_manage` tooling. It can be used to mark strings that may have a valid translation in transifex, but we want to keep the default untranslated for some reason (a name perhaps?)